site/tests/app/controllers/admin/LibraryManageControllerTester.php

```diff
@@ -26,11 +28,12 @@ class LibraryManageControllerTester extends BaseUnitTest {
     /** @var LibraryManageController */
     protected $controller;
 
-    protected function createConfigWithLibrary(bool $enabled = true, string $location = 'library location') {
+    protected function createConfigWithLibrary(bool $enabled = true, string $location = 'library location', int $access_level = User::LEVEL_SUPERUSER) {
```

Comment:
This should have a lower default access level so that the tests can pass. Only pass a higher access level for the authorization test.
```suggestion
    protected function createConfigWithLibrary(bool $enabled = true, string $location = 'library location', int $access_level = User::LEVEL_USER) {
```